### ISSUE TITLE
Grunt task to generate css/js style guide.

### ISSUE THREAD

**Issue by ethyde** [Bijoolabs/bijoo-start#12]
**Css Style guide**
[PostCss mdcss](https://github.com/jonathantneal/mdcss)
[PostCss Style Guide](https://github.com/morishitter/postcss-style-guide)
or 
[SC5 style Guide](https://github.com/SC5/sc5-styleguide)

**JsDoc Style guide**
[JSDoc](https://github.com/jsdoc3/jsdoc)
[DokkerJS](http://dokkerjs.com/)

**Comment by benoitalix** [comment]
Good idea for the styles part. Let's test with PostCss mdcss to see how interesting it is !

Regarding to the JsDoc styleguide, I think it would be great on JS oriented projects but boring for small and medium JS projects.

**Comment by ethyde** [comment]
Ok, I'm working on it !

But, for JS, I think, if we start to use requireJS, with modularity in mind, even for small project, it's a good habit to start to make good docs.
With a 1 single page we can have 4 differents modules : auto-complete/suggest, carrousel/slider, pop-in, and any ohter animation lib like TweenMax/Velocity, etc.
